### ISSUE TITLE
The GPU Memory-Usage is not changed

### ISSUE THREAD

**Issue by szhang963** [HealthML/self-supervised-3d-tasks#10]
Hi! Thanks for your code!  
I run your code successfully. However, I found the GPU Memory-Usage is very low (138M), though I adjusted the batch size in the "*.json" file. This caused the very low run speed. Could you give me some help？
Thanks in advance.

**Comment by aihamtaleb** [comment]
Hi,
Thanks for your interest. Are you sure you are using the right versions of Cuda and Tensorflow?

**Comment by szhang963** [comment]
Hi,
My environment is:
CUDA 10.1
tensorflow-gpu    2.0.0

**Comment by szhang963** [comment]
Hi,
Thanks for your mention. I have solved this problem by running this command ```conda install tensorflow-gpu=2.1```. 
However, I found the pre-train output logs in pancreas dataset include 'accuracy' and 'val_loss', 'val_accuracy' value. These metrics were evaluated for the accuracy in the upstream task?

**Comment by aihamtaleb** [comment]
Correct, when pretraining, these metrics are computed only on the upstream task. You can specify the dataset split for validation on the upstream task.

**Comment by szhang963** [comment]
Thanks for your answer.
